python/lsst/ts/ataos/ataos_csc.py

```diff
@@ -247,6 +265,36 @@ def mcs(self) -> Remote:
     def hexapod(self) -> Remote:
         return self.atcs.rem.athexapod
 
+    @property
+    def temperature(self) -> float:
+        if self._temperature_data is not None:
+            if (
+                utils.current_tai() - self._temperature_data.private_sndStamp
+                > self.max_telemetry_age
+            ):
+                self.log.warning(
+                    "Temperature telemetry too old, using fallback temperature."
+                )
+                return self.fallback_temperature
+            else:
+                # Return Truss temperature which is the second item.
+                return self._temperature_data.temperatureItem[
+                    self.temperature_item_index
```

Comment:
this attribute is not define in the `__init__` function. Make sure this is defined there with an appropriate initialization value.